modules/administration-guide/pages/configuring-editors-definitions.adoc

```diff
@@ -0,0 +1,179 @@
+:_content-type: PROCEDURE
+:description: Configuring editors definitions
+:keywords: administration guide, configuring, dashboard, editors
+:navtitle: Configuring editors definitions
+:page-aliases: installation-guide:configuring-editors-definitions.adoc
+
+[id="configuring-editors-definitions"]
+= Configuring editors definitions
+
+This procedure describes how to configure {prod-short} editor definitions.
+
+.Prerequisites
+
+* An active `{orch-cli}` session with administrative permissions to the {orch-name} cluster. See {orch-cli-link}.
+
+.Procedure
+
+. Create the `my-editor-definition-devfile.yaml` YAML file with the editor definition configuration.
++
+Ensure to provide the actual values for `publisher` and `version` under `metadata.attributes`.
+They are used to construct the editor id along with editor name in the following format `publisher/name/version`.
++
+Below you can find the supported values, including optional ones:
++
+[source,yaml,subs="+quotes,+attributes"]
+----
+# Version of the devile schema
+schemaVersion: 2.2.2
+# Meta information of the editor
+metadata:
+  # (MANDATORY) The editor name
+  name: EditorName
+  displayName: Display Name
+  description: Run Editor Foo on top of Eclipse Che
+  # (OPTIONAL) Array of tags of the current editor. Tech-Preview means it's considered experimental and is not recommended for production environments. While it can include new features and improvements, it may still contain bugs or undergo significant changes before reaching a stable version.
```

Comment:
```suggestion
  # (OPTIONAL) Array of tags of the current editor. The Tech-Preview tag means the option is considered experimental and is not recommended for production environments. While it can include new features and improvements, it may still contain bugs or undergo significant changes before reaching a stable version.
```